brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java

```diff
@@ -104,16 +105,24 @@ public void decodeManifest(File outDir) throws AndrolibException {
         if (hasManifest()) {
             if (mConfig.decodeResources == Config.DECODE_RESOURCES_FULL ||
                 mConfig.forceDecodeManifest == Config.FORCE_DECODE_MANIFEST_FULL) {
+                ResTable resTable = getResTable();
```

Comment:
Is there any reason to making an intermediate variable for the res table? Its already on `mResTable`?

Reply:
resTable used in the 2 place:

1. in the `decodeManifest(resTable, ...`
2. in the `adjustPackageManifest(resTable, ...`

Reply:
Sorry what I mean:

 * You have `getResTable` which gives you `mResTable` on the class itself (`ResourcesDecoder`) while loading packages if not loaded yet.
 * You set the output of that to an intermediate value `resTable` and pass onward to functions.
 
 So we have a class property `mResTable` and an intermediate `resTable` and I'm confused of the introduction of the intermediate one.